run.py

```diff
@@ -454,12 +454,15 @@ def calculate_average_read_length(file_path):
             rrna_reads_dict[input_filename] / subset_reads_dict[input_filename]
             for input_filename in total_reads_dict
         ]
-
         # Use the total number of reads for each input as weights
         weights = list(total_reads_dict.values())
-        weighted_rrna_fraction = np.average(
-            fractions_rrna_in_subset, weights=weights
-        )
+        try:
+            weighted_rrna_fraction = np.average(
+                fractions_rrna_in_subset, weights=weights
+            )
+        except ZeroDivisionError:
+            weighted_rrna_fraction = 0
+            print("Sample doesn't contain ribosomal reads.")
```

Comment:
Is this actually what a ZeroDivisionError means here? I'd expect it to happen when there were zero reads, not zero ribosomal reads.

Reply:
@lennijusten can you say more? I'm not super familiar with this code.

Reply:
`weights` is calculated using the total reads in each subset where subsets are subsets of the different adapter trimming output files. I would expect weights to sum to zero if there were zero reads. 

So @jeffkaufman is right. If we get the `ZeroDivisionError` it means we have zero total reads and hence also zero rRNA reads.

Reply:
Ok cool, I'll revisit this (might also check in with you in person @lennijusten)